templates/index.html

```diff
@@ -367,45 +414,70 @@ <h2 class="u-no-margin--bottom"><strong>Accelerating innovation</strong></h2>
         <h3 class="p-text--small-caps">Cloud</h3>
       </div>
       <div class="col-9 col-medium-4">
-        <p class="p-heading--2">Streamline your workflows and get the scalability you need with <a href="/partners/public-cloud">Ubuntu across clouds</a>.</p>
+        <p class="p-heading--2">Streamline your workflows and get the scalability you need with <a
+            href="/partners/public-cloud">Ubuntu across clouds</a>.</p>
         <div class="row">
           <div class="p-logo-section">
             <div class="p-logo-section__items">
               <div class="p-logo-section__item">
-                  {{ image (
-                    url="https://assets.ubuntu.com/v1/5afa90c1-google-cloud-logo.png",
-                    alt="",
-                    width="696",
-                    height="474",
-                    hi_def=True,
-                    loading="lazy",
-                    attrs={"class": "p-logo-section__logo"}
-                    ) | safe
-                  }}
+                {{ image (
+                url="https://assets.ubuntu.com/v1/5afa90c1-google-cloud-logo.png",
+                alt="",
+                width="696",
+                height="474",
```

Comment:
doesn't have the same order of logos to the design (relevant at all...?)

Reply:
indeed, good catch.

Reply:
To be honest, I'm a little hesitant to fix this since it's been like for a while on production. It this was an issue we would have been notified. I guess it doesn't make any difference? I will ping the stakeholders about this anyway, but let's not block this PR because of that.